### ISSUE TITLE
[Martine] - Reduce and/or reorder the scenes

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-people-use-web-videos#13]
I think this latest version (on May 2nd) is teetering on getting too long & too detailed. If nothing else, we should cut and not add at this point. 

However, I also think it jumps around a bit too much. I wonder if a flow around communication might help. It was easier to edit the text itself than make individual suggestions so my rewrite draft is below. Essentially, I wonder if it would be better to chunk the information more. 

I also want to suggest being explicit about what happens when we point out something bad (that is, rather than just leave things hanging, explicitly say how it affects the person and/or what they do to work around it). 

****
Suggested rewrite (just to have the conversation about ordering the content as well as how to maybe cut back on some words/ideas):

_[Audio updates - Move some intro stuff around a bit]_
Hello! I’m Martine. I have been deaf since birth. I can hear some sounds but not enough to hear speech, so sign language is my primary language and preferred communication method. I’m also proudly a 62 year old college student! I recently enrolled in online courses at my local university. 
_[Suggested Scene - Martine talking to camera]_

_[Audio - Schedule interpreters + pinning them; added a bit more about what happens when you can't pin; tried to succinctly talk about scheduling interpreters]_
Through the school I can book sign language interpreters for scheduled lectures and meetings. The interpreters sign what others are saying aloud, and speak aloud my signing for others to hear. This works best when I can pin (or lock-in) the speaker and interpreter windows on the video conferencing apps. Unfortunately, when apps don’t have this functionality I have to find workarounds like using my phone to show my interpreter.
_[Suggested Scene(s) - Martine watching a lecture, pinning a window, and then sadly watching with interpreter on phone]_

_[Audio - Talk about captions features all at once; can trim but seemed to make sense to talk about pre-recorded versus auto separately]_
When interpreters aren’t available, the next best option for me is captions, for example when a video is pre-recorded. Captions work *especially* well when they include who is speaking, point out the important audio sounds, and are timed with the spoken audio without being too long or fast. I also love when I can customize the fonts and position on the screen.
_[Suggested scene - show good pre-recorded captions]_

_[Audio - Talk about auto captions all at once; can trim any parts of this down]_
When events are live but unscheduled, like a quick meeting with my classmates, we often use automatic captions. While I appreciate that technology a lot, it still sometimes causes issues for me. For instance, they don’t show who’s speaking, so when people talk back-to-back it can be hard to follow. My classmates are great about it though — they make sure to talk clearly, raise their hand to take turns talking, and check in with me to make sure I’m following.
_[Suggested scene(s) - confusing auto captions with no punctuation then a turn taking convo with good caption breaks]_

_[Audio - talk about lip reading and emphasize it’s not a primary function]_
Sometimes I might also lip read when I can clearly see a person’s face. I do this a lot for social events, like when my [friends/classmates/family] hosted a virtual game night. However, it gets tiring to do that for too long, and it’s not reliable for important stuff like my classwork.
_[Suggested scene - group call, lots of laughing, maybe a card game?]_

_[Audio - explicitly transition to written text since it may not be top-of-mind for many when they think about deaf folks]_
So that’s a lot about communicating on video and in-person but one last point is about writing. I can understand written English; however, at some point if it’s too complex it actually gets hard to understand. People sometimes don’t realize American Sign Language has different grammar than English so I’m actually translating when I’m reading. 
_[Suggested scene - back to Martine talking to us; we only showed her at the beginning and end but maybe she can appear a few times in the middle too]_

_[Audio - describe good/bad text; can be combined with text above as well - I think we can pair down the speech overall to be a bit more succinct]_
To help with that, I like when text is in reasonable-sized paragraphs, and broken up with headings and lists when possible. I also better understand language this is direct and to the point. When sentences get too long and written in passive voice or complex language, I can’t follow it.
_[Suggested scene - show either good and/or bad text on paper with satisfied and/or confused look on Martine’s face]_

_[Audio - wrap-up; just a suggestion but I like the idea of a funny/smiling send off]_
So that’s a bit about me and how, when people like me are considered, I can accomplish great things like getting my degree! Speaking of which, I gotta get back to my homework. See ya!
_[Suggested scene - Martine saying bye to us!]_

**Comment by nitedog** [comment]
Many thanks, Michelle! I think there are some excellent thoughts in your suggested rewrite. At the same time, it actually increases the word count for the audio track (what Martine says, which mainly impacts the duration of the video) from 405 words to 453 words (~12% more), rather than to reduce the length. I've been wrestling with saying things succinctly yet sufficiently explanatory. During today's meeting, let's explore which aspects we can extract from your proposal to improve the script.

**Comment by jade-mc** [comment]
A follow up from yesterday. 
I've had a scout round online, you can use the term 'written language', (but it does have overlaps with representing a spoken/gestural language.) If you don't use this, then I'd suggest 'reading and writing' which would capture all of that comprehension and open the text up to deal with more situations. 

As for 'signed' vs 'sign' language, we should go with 'sign'. Signed is used in some research contexts, but it aligns it more with the spoken modality, this is seen by many as taking it away from the values and identity of the Deaf community. 

Remind me again why you want to remove the reference to English/ASL? I actually think it might benefit the text if you leave it in. I know that might appear to make it less global, but if you stick in 'I'm from the USA so I use ASL' then that means you'd be able to keep 'written English' in there.

**Comment by nitedog** [comment]
I believe to have addressed this thread in the [latest edit of 13th May](https://deploy-preview-16--wai-people-use-web-videos.netlify.app/people-use-web/videos/stories/martine/) but please re-open if you disagree.
